app/javascript/features/message/message.js

```diff
@@ -0,0 +1,19 @@
+import React, {useEffect} from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { fetchMessage } from './messageSlice';
+
+
+export default function Message() {
+  const message = useSelector(state => state.message.message);
+  const dispatch = useDispatch();
+
```

Comment:
- [ ] Please change this component name to `Greeting` as requested in the project requirements.